### PR TITLE
Create validations javascript.

### PR DESCRIPTION
**Pull request by awong-dev**
Stub out the file so our group can collaboratively hack.

### CODE REVIEW

File: _health-care/_js/_utils/validations.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function isValidSSN(value) {
+  return /\d\d\d-\d\d-\d\d\d\d/.test(value);
```
Comment (akainic) [inline _health-care/_js/_utils/validations.js]: Is there a reason we don't want to include this way: `/^\d{3}-\d{2}-\d{4}$/`?

Comment (awong-dev) [inline _health-care/_js/_utils/validations.js]: ...derp... I suck at regexps?

File: spec/javascripts/health-care/utils/validations.spec.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import Validations from '../../../../_health-care/_js/_utils/validations.js';
```
Comment (akainic) [inline spec/javascripts/health-care/utils/validations.spec.js]: You don't have the `Validations` class yet, right?

Comment (awong-dev) [inline spec/javascripts/health-care/utils/validations.spec.js]: I don't expect to ever have such a class. :P

`import Validations from 'blah'`  conceptually means
`const Validations = getDefaultExportFrom(blah);`

I can name the import whatever I want.

Comment (akainic) [inline spec/javascripts/health-care/utils/validations.spec.js]: Oh cool!

**Comment by akainic** [comment]
LGTM!

**Pull request merged**